packages/flutter_tools/lib/src/vmservice.dart

```diff
@@ -897,6 +897,25 @@ class HeapSpace extends ServiceObject {
   }
 }
 
+// A function, field or class along with its source location.
+class ProgramElement {
+  String qualifiedName;
+  Uri uri;
+  int line;
+  int column;
+
+  ProgramElement(this.qualifiedName, this.uri, this.line, this.column);
+
+  @override
+  String toString() {
+    if (line == null) {
+      return "$qualifiedName ($uri)";
```

Comment:
could the URI be made relative to the current path? It'd be much nicer to see "lib/main.dart" rather than "file:///usr/local/google/home/rmacnak/Flutter/flutter/examples/flutter_gallery/lib/main.dart".

Reply:
Now using relative paths for things inside the project directory.